README.md

```diff
@@ -187,7 +187,7 @@ The function will receive these parameters:
 | `laneBag`    | A bag with some helper functions to work with the lane |
 
 ##### `laneBag`
-| function     | Description                                            |
```

Comment:
I'd rather not to change this now. This is the pattern to the entire doc. You could open another to PR to do that.